resources/postgres/postgres.go

```diff
@@ -154,6 +171,23 @@ func initDB(pgBinDir, pgDataDir, pgUser string) error {
 		"LC_ALL=en_US.UTF-8",
 		"LC_CTYPE=en_US.UTF-8",
 	)
+	currentUser, err := user.Current()
+	if err != nil {
+		return fmt.Errorf("failed to get current user, error: %w", err)
+	}
+	// If user runs bytebase as root user, we will attempt to run initdb as user `bytebase`.
```

Comment:
Worth pointing to the official doc

Reply:
Also the logic seems to chmod of the data directory after initdb instead of running initdb as root user.

Same in pg_ctl above